crates/astria-core/src/sequencerblock/v1alpha1/block.rs

```diff
@@ -1439,6 +1467,8 @@ enum DepositErrorKind {
     IncorrectRollupIdLength(#[source] IncorrectRollupIdLength),
     #[error("the `asset` field could not be parsed")]
     IncorrectAsset(#[source] asset::ParseDenomError),
+    #[error("error converting to `TransactionId`")]
```

Comment:
Don't leak the type that you are converting to here; the following is much more descriptive to a reader:
```suggestion
    #[error("field `id_of_source_transaction` was invalid")]
```